Source/Csla.Shared/ALC/ALCManager.cs

```diff
@@ -0,0 +1,112 @@
+﻿#if NET5_0_OR_GREATER
+
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Runtime.Loader;
+
+namespace Csla.ALC
+{
+    /// <summary>
+    /// Provides common access to "AssemblyLoadContext" logic, which is used for "Plugins" concept of .NET 5+
+    /// </summary>
+    public static class ALCManager
```

Comment:
Type should be named `AlcManager`

Reply:
If you want so, named like this because "AssemblyLoadContext" - three words, but "Alc" seems like shortcut of one word, like "Alcohol" :D

Reply:
Perhaps `AssemblyLoadContextManager` is the best then - avoid confusing acronyms and make the code more readable overall.

Especially if you like my thought about a `Csla.Runtime` namespace, it would help be a little self-documenting.